src/server.rs

```diff
@@ -548,6 +551,7 @@ impl ClientsInfo {
 
 pub(super) struct ClientInfo {
     id: ClientId,
+    just_connected: bool,
```

Comment:
Can we throttle init messages? They use reliable channel and Renet doesn't expose acks API :(
And messages contain only changes since the previous tick, so even if client abuses it, we don't send extra data (but transport will resend misses).

Reply:
Throttling init messages using resend time sounds like a good idea.

Then let's keep `bool` for now until we decide on approach (will be more clear when we start implementing it).